File: app/src/main/java/com/example/food_app/activity/BaseActivity.java
package com.example.food_app.activity;

import android.os.Bundle;
import android.view.Window;
import android.view.WindowManager;

import androidx.appcompat.app.AppCompatActivity;

import com.example.food_app.R;

public class BaseActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_base);

        Window w = getWindow();
        w.setFlags(WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS, WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS);

    }
}

File: app/src/main/java/com/example/food_app/activity/BaseActivity.kt
package com.example.food_app.activity

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.food_app.R
import com.google.firebase.database.FirebaseDatabase

open class BaseActivity : AppCompatActivity() {
    var database: FirebaseDatabase? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        database = FirebaseDatabase.getInstance()
        setContentView(R.layout.activity_base)
    }
}